cdc/src/main/java/com/hazelcast/jet/contrib/cdc/CdcSources.java

```diff
@@ -0,0 +1,195 @@
+        properties.put("connector.class", "io.debezium.connector.sqlserver.SqlServerConnector");
+
+        properties.putIfAbsent("database.history", HazelcastListDatabaseHistory.class.getName());
+        properties.putIfAbsent("database.history.hazelcast.list.name", name);
+
+        properties.putIfAbsent("tombstones.on.delete", "false");
+
+        return KafkaConnectSources.connect(properties,
+                record -> {
+                    String keyJson = Values.convertToString(record.keySchema(), record.key());
+                    String valueJson = Values.convertToString(record.valueSchema(), record.value());
+                    return new ChangeEventRelationalImpl(keyJson, valueJson, OBJECT_MAPPER);
+                }
+        );
+    }
+
+    /**
+     * Creates a CDC source that streams change data from your
+     * MongoDB database to the Hazelcast Jet pipeline.
+     *
+     * @param properties configuration object which holds the configuration
+     *                   properties of the connector.
+     * @return a source to use in {@link com.hazelcast.jet.pipeline.Pipeline#readFrom(StreamSource)}
+     */
+    public static StreamSource<ChangeEvent> mongodb(String name, Properties properties) {
+        properties = copy(properties);
+
+        /* Used internally as a unique identifier when recording the
+        oplog position of each replica set. Needs to be set. */
+        checkSet(properties, "mongodb.name");
+
+        properties.put("name", name);
+        properties.put("connector.class", "io.debezium.connector.mongodb.MongoDbConnector");
+
+        properties.putIfAbsent("database.history", HazelcastListDatabaseHistory.class.getName());
+        properties.putIfAbsent("database.history.hazelcast.list.name", name);
+
+        /* When running the connector against a sharded cluster, use a
+        value of tasks.max that is greater than the number of replica
+        sets. This will allow the connector to create one task for each
+        replica set, and will let Kafka Connect coordinate, distribute,
+        and manage the tasks across all of the available worker
+        processes.*/
+        properties.putIfAbsent("tasks.max", 1);
+
+        /*Positive integer value that specifies the maximum number of
+        threads used to perform an intial sync of the collections in a
+        replica set.*/
+        properties.putIfAbsent("initial.sync.max.threads", 1);
+
+        properties.putIfAbsent("tombstones.on.delete", "false");
+
+        return KafkaConnectSources.connect(properties,
+                record -> {
+                    String keyJson = Values.convertToString(record.keySchema(), record.key());
+                    String valueJson = Values.convertToString(record.valueSchema(), record.value());
+                    return new ChangeEventMongoImpl(keyJson, valueJson);
+                }
+        );
+    }
+
+    private static void checkSet(Properties properties, String key) {
```

Comment:
We can replace this with `com.hazelcast.internal.util.Preconditions#checkHasText`